buildpack/buildpack.go

```diff
@@ -18,14 +18,17 @@ package buildpack
 
 import (
 	"fmt"
-	"path/filepath"
-
 	"github.com/buildpack/libbuildpack/buildpack"
 	"github.com/cloudfoundry/libcfbuildpack/logger"
 	"github.com/mitchellh/mapstructure"
+	"path/filepath"
 )
 
-const cacheRoot = "dependency-cache"
+const (
+	cacheRoot            = "dependency-cache"
+	DEPENDENCIES         = "dependencies"
+	DEFAULT_DEPENDENCIES = "default_dependencies"
```

Comment:
This should be, as the PR mentions, `default_versions` rather than `default_dependencies`.  The map describes a collection of ids to versions, not ids to dependencies.